transform/transformers/common_software/pck_transformer.py

```diff
@@ -265,12 +363,18 @@ def preprocess_comments(self):
 
     def calculate_total_playback(self):
         """
+        For QCAS:
         Calculates the total value for both acquisitions and proceeds from disposals for machinery and equipment section
```

Comment:
For discussion: Starting to think that we should adopt a strategy pattern approach for our continuing growth in surveys . Not for this PR . But in this case instead of calculate_total_playback with if statements within in it , we set a calculate_total_playback strategy (and other strategies) when we identify the survey and simply execute that strategy. That  way if we end up with many variations we dont care , we avoid the 'ifs' inside calculate_total_playback.

Reply:
You're not wrong.  There's a card in on the board to look at how it would be sensible to tidy up parts of this repo as it's grown...organically as more stuff has been added to it.

For stocks in particular, I would've made its own transformer if I'd been told upfront that we'd have to calculate the totals!